src/python/national_voter_file/us_states/vt/transformer.py

```diff
@@ -0,0 +1,272 @@
+    col_type_dict['PRECINCT_SPLIT'] = set([str, type(None)])
+    col_type_dict['PRECINCT'] = set([str, type(None)])
+
+    #### Contact methods #######################################################
+
+
+    #### Demographics methods ##################################################
+
+    def extract_birthdate(self, input_columns):
+        """
+        Inputs:
+            input_columns: name or list of columns
+        Outputs:
+            Dictionary with following keys
+                'BIRTHDATE'
+        """
+        return {
+            'BIRTHDATE': self.convert_date(date_str=input_columns['Year of Birth'], date_format='%Y'),
+            'BIRTHDATE_IS_ESTIMATE': 'Y'
+        }
+
+
+    #### Address methods #######################################################
+
+    def extract_registration_address(self, input_dict):
+        """
+        Relies on the usaddress package.
+
+        Call the self.convert_usaddress_dict() method on the output of
+        usaddress.tag. We provide example code in the method to make this clear.
+
+        Inputs:
+            input_dict: dictionary of form {colname: value} from raw data
+        Outputs:
+            Dictionary with following keys
+                'ADDRESS_NUMBER'
+                'ADDRESS_NUMBER_PREFIX'
+                'ADDRESS_NUMBER_SUFFIX'
+                'BUILDING_NAME'
+                'CORNER_OF'
+                'INTERSECTION_SEPARATOR'
+                'LANDMARK_NAME'
+                'NOT_ADDRESS'
+                'OCCUPANCY_TYPE'
+                'OCCUPANCY_IDENTIFIER'
+                'PLACE_NAME'
+                'STATE_NAME'
+                'STREET_NAME'
+                'STREET_NAME_PRE_DIRECTIONAL'
+                'STREET_NAME_PRE_MODIFIER'
+                'STREET_NAME_PRE_TYPE'
+                'STREET_NAME_POST_DIRECTIONAL'
+                'STREET_NAME_POST_MODIFIER'
+                'STREET_NAME_POST_TYPE'
+                'SUBADDRESS_IDENTIFIER'
+                'SUBADDRESS_TYPE'
+                'USPS_BOX_GROUP_ID'
+                'USPS_BOX_GROUP_TYPE'
+                'USPS_BOX_ID'
+                'USPS_BOX_TYPE'
+                'ZIP_CODE'
+        """
+
+        columns = [
+            'Legal Address Line 1',
+            'Legal Address Line 2',
+            'Legal Address City',
+            'Legal Address State',
+            'Legal Address Zip'
+        ]
+
+        mail_str = ' '.join([input_dict[x] for x in columns if input_dict[x] is not None])
+        usaddress_dict, usaddress_type = self.usaddress_tag(mail_str)
+
+        raw_dict = {
+            'RAW_ADDR1': input_dict['Legal Address Line 1'],
+            'RAW_ADDR2': input_dict['Legal Address Line 2'],
+            'RAW_CITY': input_dict['Legal Address City'],
+            'RAW_ZIP': input_dict['Legal Address Zip'],
+        }
+
+        for r in ['RAW_ADDR1', 'RAW_ADDR2']:
+            if not raw_dict[r].strip():
+                raw_dict[r] = '--Not provided--'
```

Comment:
We're moving to using `None` instead of any magic strings here, so these three lines can be removed. (It's a recent change as of #176)